packages/cli/src/endo.js

```diff
@@ -590,11 +591,20 @@ export const main = async rawArgs => {
     });
 
   program
-    .command('reset')
+    .command('purge')
+    .option('-f, --force', 'skip the confirmation prompt')
     .description('erases persistent state and restarts if running')
-    .action(async _cmd => {
-      const { reset } = await import('@endo/daemon');
-      await reset();
+    .action(async cmd => {
+      const { force } = cmd.opts();
+      const doPurge =
+        force ||
+        /^y(es)?$/u.test(
+          await prompt('Are you sure you want to erase all state? (y/n)'),
```

Comment:
Might be worth noting that this is not reversible and will permanently sever any connections to previously known peers. The formula identifiers are (or soon will be, working copy) rooted at a random seed that is regenerated after a purge.

Reply:
Like this, perhaps?
```suggestion
          await prompt(
            'Are you sure you want to erase all state? This irreversible action will permanently sever all peer connections and destroy all formulae. Continue? (y/n)'
          ),
```